_posts/2025-02-01-lts-cadence.adoc

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: 'Quarkus LTS - Release Cadence Explained'
```

Comment:
```suggestion
title: 'Quarkus LTS - New Release Cadence Explained'
```

reads as if explaining existing release cadence but we are announcing something new